### DISCUSSION TITLE
Using node js and socket.io

### DISCUSSION THREAD

**Discussion by iLikeGrape** [socketio/socket.io#5233] [Q&A] [answered]
Hello!
I am a new dev trying to learn how things work. I have heard of these things, but don't know what they do or what they mean. Every tutorial I find moves to fast and talks about things I don't understand. Here is the extent of my knowledge: ilikegrape.netlify.app

Does anyone know a good tutorial for these things? Ideally it would be with javascript and html. Thanks!

**Comment by darrachequesne** [comment, marked as answer]
Hi!

Please take a look at our tutorial: https://socket.io/docs/v4/tutorial/introduction

It starts from the beginning (html, express) and then makes a tour of the library. Please ping me if something is not clear enough.

**Reply by iLikeGrape** [reply]
Ok thanks!
